setup.py

```diff
@@ -1,56 +1,79 @@
-        "Programming Language :: Python",
-        "Programming Language :: Python :: 2.7",
-        "Programming Language :: Python :: 3.4",
-        "Programming Language :: Python :: 3.5",
-        "Programming Language :: Python :: 3.6",
-        "Operating System :: OS Independent",
-        "Topic :: Scientific/Engineering :: Medical Science Apps.",
-        "Topic :: Scientific/Engineering :: Physics",
-        "Topic :: Software Development :: Libraries",
-    ],
-    long_description="""
-    pydicom is a pure python package for parsing DICOM files.
-    DICOM is a standard (http://medical.nema.org) for communicating
-    medical images and related information such as reports
-    and radiotherapy objects.
-
-    pydicom makes it easy to read these complex files into natural
-    pythonic structures for easy manipulation.
-    Modified datasets can be written again to DICOM format files.
-    See the `Getting Started <http://pydicom.readthedocs.org/en/latest/getting_started.html>`_
-    page for installation and basic information, and the
-    `Pydicom User Guide <http://pydicom.readthedocs.org/en/latest/pydicom_user_guide.html>`_ page
-    for an overview of how to use the pydicom library.
-    """,
-    **extra
-)
+from setuptools import setup, find_packages
+from pydicom import __version__
+
+
+description = "Pure python package for DICOM medical file reading and writing"
+
+long_description = """
+
+pydicom
+=======
+
+pydicom is a pure python package for parsing DICOM files. DICOM is a standard
+(http://medical.nema.org) for communicating medical images and related
+information such as reports and radiotherapy objects.
+
+pydicom makes it easy to read these complex files into natural pythonic
+structures for easy manipulation.  Modified datasets can be written again to
+DICOM format files. See the `Getting Started
+<http://pydicom.readthedocs.org/en/latest/getting_started.html>`_ page for
+installation and basic information, and the `Pydicom User Guide
+<http://pydicom.readthedocs.org/en/latest/pydicom_user_guide.html>`_ page for
+an overview of how to use the pydicom library.
+"""
+
+CLASSIFIERS = [
+    "License :: OSI Approved :: MIT License",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Healthcare Industry",
+    "Intended Audience :: Science/Research",
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3.4",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
+    "Operating System :: OS Independent",
+    "Topic :: Scientific/Engineering :: Medical Science Apps.",
+    "Topic :: Scientific/Engineering :: Physics",
+    "Topic :: Software Development :: Libraries"]
+
+KEYWORDS = "dicom python medical imaging"
+
+NAME = "pydicom"
+AUTHOR = "Darcy Mason and contributors"
+AUTHOR_EMAIL = "cdsupsay@gmail.com"
```

Comment:
> The appveyor should have the same configuration matrix than Travis, but I would merge as it is. (I maintain my +1)

Not exactly since we are testing the ubuntu package in particular. And I let you the `pypy` install with the VS code in 32 bits :)